src/test/java/SetTest.java

```diff
@@ -0,0 +1,46 @@
+import java.util.Set;
+import java.util.HashSet;
+import org.junit.jupiter.api.*;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+import static org.assertj.core.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.*;
+
+
+class SetTest {
+
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    @DisplayName("Set의 크기를 구하는 테스트")
+    public void size() {
+        int size = numbers.size();
+        assertThat(size).isEqualTo(3);
+    }
+
+    @ParameterizedTest
+    @ValueSource(ints = {1,2,3})
+    @DisplayName("Set의 값이 존재하는지 확인")
+    public void isContains(int number) {
+        assertTrue(numbers.contains(number));
```

Comment:
```suggestion
        assertThat(numbers.contains(number)).isTrue();
```
이렇게 하면 가독성이 조금더 좋아질 거 같아요.😀